packages/@angular/cli/commands/generate.ts

```diff
@@ -26,17 +26,12 @@ const GenerateCommand = EmberGenerateCommand.extend({
     // map the blueprint name to allow for aliases
     rawArgs[0] = mapBlueprintName(rawArgs[0]);
 
-    if (rawArgs[0] !== '--help' &&
+    if (['--help', '-h'].indexOf(rawArgs[0]) === -1 &&
       !fs.existsSync(path.join(__dirname, '..', 'blueprints', rawArgs[0]))) {
       SilentError.debugOrThrow('@angular/cli/commands/generate',
         `Invalid blueprint: ${rawArgs[0]}`);
     }
 
-    if (!rawArgs[1]) {
-      SilentError.debugOrThrow('@angular/cli/commands/generate',
```

Comment:
I think we still need an error like this to be shown, otherwise `ng generate` (without a blueprint) will just say `Invalid blueprint`, right?